azurerm/internal/services/batch/resource_arm_batch_pool.go

```diff
@@ -372,6 +372,14 @@ func resourceArmBatchPool() *schema.Resource {
 					},
 				},
 			},
+			"metadata": {
+				Type:     schema.TypeMap,
+				Optional: true,
+				ForceNew: false,
```

Comment:
This property can't be updated in place? 🤔

Reply:
From my tests you can just change the metadata without forcing a new resource.

Reply:
in which case we can remove this:

```suggestion
```

since `false` is the default value for a boolean field - this is impled/superfluous

Reply:
done